pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: flutter_form_builder
 description: This package helps in creation of forms in Flutter by removing the boilerplate code, reusing validation, react to changes, and collect final user input.
-version: 9.4.1
+version: 9.4.2
```

Increase de minor is the right way here, because the changes isn't a hotfix
```suggestion
version: 9.5.0
```